tools/github/github_utils.py

```diff
@@ -0,0 +1,81 @@
+"""GitHub Utilities"""
```

Comment:
Do you plan to refactor scripts like https://github.com/pytorch/pytorch/blob/f3c3d12efb363f61ca9d68ed9e55f345f1a7e388/tools/stats/upload_external_contrib_stats.py#L32 a bit to use this utils module to avoid code duplication?  These functions all look very familiar to me somehow :)